SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -223,6 +218,12 @@ void USpatialGameInstance::HandleOnConnected()
 	WorkerConnection->OnEnqueueMessage.AddUObject(SpatialLatencyTracer, &USpatialLatencyTracer::OnEnqueueMessage);
 	WorkerConnection->OnDequeueMessage.AddUObject(SpatialLatencyTracer, &USpatialLatencyTracer::OnDequeueMessage);
 #endif
+
+	OnSpatialConnected.Broadcast();
+}
+
+void USpatialGameInstance::CleanupCachedLevelsAfterConnection()
```

Comment:
This processing needed to be moved until after the LBStrategy had been assigned a virtual worker ID.

Reply:
Can we combine the logic of this function and GSM::TriggerBeginPlay? They are both called in the same place (after server is ready) and both iterate over all Actors in the world.

Reply:
hmmm, follow-up PR? Once ActorGroupManager is removed, we might be able to move all of this code out of the SpatialGameInstance, but I think it would be better to isolate those changes.

Reply:
SGTM, can we whack in a TODO just in case we forget?

Reply:
Created an UNR ticket, which is easier to track than a TODO.  :)